### DISCUSSION TITLE
Rover on Ariakit

### DISCUSSION THREAD

**Discussion by theo-mesnil** [ariakit/ariakit#2572] [Questions] [answered]
Hi, 

On reakit we used Rover for our [Pagination](https://www.welcome-ui.com/components/pagination) components.
We migrate to ariakit and i would like to know how can we do the same thing with ariakit ? 

Thanks :)

**Comment by diegohaz** [comment, marked as answer]
Hey!

I'd recommend just using native tabbable elements like links. But [Composite](https://ariakit.org/components/composite) is the component you're looking for. It's also available on Reakit: https://reakit.io/docs/composite

**Reply by theo-mesnil** [reply]
Ok thanks @diegohaz, i'll try :)
